core/src/main/java/org/apache/calcite/sql2rel/RelFieldTrimmer.java

```diff
@@ -1055,15 +1055,19 @@ public TrimResult trimFields(
     final ImmutableBitSet.Builder inputFieldsUsed =
         aggregate.getGroupSet().rebuild();
     // 2. agg functions
+    int index = aggregate.getGroupCount();
```

Comment:
nit: I think it would be more clear if this variable was named something like `int aggCallIndex`